src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -170,7 +170,6 @@ namespace winrt::TerminalApp::implementation
 
         const auto canDragDrop = CanDragDrop();
 
-        _tabRow.PointerMoved({ get_weak(), &TerminalPage::_RestorePointerCursorHandler });
```

Comment:
Moved into `TerminalControl` as a different workaround is needed for this functionality.